packages/fxa-content-server/tsconfig.json

```diff
@@ -9,13 +9,13 @@
         // "lib": [],                             /* Specify library files to be included in the compilation. */
         // "allowJs": true,                          /* Allow javascript files to be compiled. */
         // "checkJs": true,                       /* Report errors in .js files. */
-        // "jsx": "preserve",                     /* Specify JSX code generation: 'preserve', 'react-native', or 'react'. */
+        "jsx": "react"                     /* Specify JSX code generation: 'preserve', 'react-native', or 'react'. */,
         "declaration": true /* Generates corresponding '.d.ts' file. */,
         // "declarationMap": true,                /* Generates a sourcemap for each corresponding '.d.ts' file. */
         // "sourceMap": true,                     /* Generates corresponding '.map' file. */
         // "outFile": "./",                       /* Concatenate and emit output to single file. */
         "outDir": "./.tscompiled/" /* Redirect output structure to the directory. */,
-        "rootDir": "./app/scripts/" /* Specify the root directory of input files. Use to control the output directory structure with --outDir. */,
+        //"rootDir": "./app/scripts/" /* Specify the root directory of input files. Use to control the output directory structure with --outDir. */,
```

Comment:
Delete?

Reply:
I left it in since all of the other options are commented out in this file